web/clusters.go

```diff
@@ -319,13 +342,15 @@ func NewClusterListHandler(clustersService services.ClustersService) gin.Handler
 		healthContainer := NewClustersHealthContainer(clusterList)
 		healthContainer.Layout = "horizontal"
 
-		page := c.DefaultQuery("page", "1")
-		perPage := c.DefaultQuery("per_page", "10")
-		pagination := NewPaginationWithStrings(len(clusterList), page, perPage) // not nice here
-		firstElem, lastElem := pagination.GetSliceNumbers()
+		count, err := clustersService.GetCount()
```

Comment:
Do we really need this `GetCount` call? Why not just get the len of the `clusterList` ?

Reply:
Ok, nevermind. We cannot use the len of the `clusterList` as it already has the filters, and we need the total number of items